src/main/java/com/aws/greengrass/deployment/exceptions/RetryableCloudDeploymentException.java

```diff
@@ -0,0 +1,39 @@
+/*
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.aws.greengrass.deployment.exceptions;
+
+import com.aws.greengrass.deployment.errorcode.DeploymentErrorCode;
+
+/**
+ * Exception for handling 5xx deployment failures.
+ */
+public class RetryableCloudDeploymentException extends DeploymentException {
+    static final long serialVersionUID = -3387516993124229948L;
+
+    public RetryableCloudDeploymentException(String message) {
+        super(message);
+    }
+
+    public RetryableCloudDeploymentException(String message, Throwable cause) {
+        super(message, cause);
+    }
+
+    public RetryableCloudDeploymentException(Throwable throwable) {
+        super(throwable);
+    }
+
+    public RetryableCloudDeploymentException(String message, DeploymentErrorCode errorCode) {
+        super(message);
+        super.addErrorCode(errorCode);
+    }
+
+    @Override
+    public RetryableCloudDeploymentException withErrorContext(Throwable t, DeploymentErrorCode errorCode) {
+        super.withErrorContext(t, errorCode);
+        return this;
+    }
```

Comment:
nit: you probably don't need these since there is no error code involved

Reply:
Remove everything that's unused